test/signhash.ts

```diff
@@ -142,4 +142,66 @@ contract('SignHash', accounts => {
       });
     });
   });
+
+  describe('#removeProof', () => {
+    it('should remove proof method', async () => {
+      const signer = accounts[0];
+      const method = 'http';
```

Comment:
That's a bit unrelated to the PR, but do you plan to have `http` and `https` as a separate methods? I'm not sure how those should be differentiated. I imagined that there should be just a `domain` method

Reply:
`https == HTTP over SSL`. When accessing a static file on a server we are using HTTP protocol in the first place. I thought about using SSL certificate as identity proof but I have no idea yet how to integrate it with our identity verification mechanism. 

I would avoid using `domain` method instead of `http` as it would collide with `dns` custom records verification which is another way to proof the domain.